Components/Optimizers/AdaptiveStochasticLBFGS/elxAdaptiveStochasticLBFGS.h

```diff
@@ -185,11 +185,11 @@ class AdaptiveStochasticLBFGS :
 
   /** Codes of stopping conditions
```

Comment:
Looks OK, but I would prefer to just remove the entire commented section, including the (redundant) `StopConditionType` definition.

Reply:
@haampie Could you possibly still amend your commit, to simply remove the entire commented section, including the StopConditionType definition?